Automattic/wp-calypso
# Domains: Update domain search empty state and input placeholder

#### Changes proposed in this Pull Request
This PR adds the new empty content state on the domain search page: when searching for a new domain, no suggestion is shown anymore. Additionally, it adds the illustration for the domains' empty content and an animated placeholder on the search bar.

#### Testing instructions
- Visit `/domains/add` page and select a domain for testing;
- Check that no domain is suggested when browsing to it;
- Check that the empty content state is shown if there's no search term input;
- Check that the search bar has a placeholder that changes the TLD with a typing animation.

#### Preview

**No search term**

![image](https://user-images.githubusercontent.com/18705930/128892546-8ed56d7b-d18a-47fc-92c5-afe89b2feeaa.png)

**Search term shorter than the minimum allowed**

![image](https://user-images.githubusercontent.com/18705930/128892597-f75c2b40-2dc9-4051-814f-065f6a5447ad.png)

**Search bar animated placeholder**

https://user-images.githubusercontent.com/18705930/128892909-e9455364-1369-4dc2-a355-ade5599f8780.mov
